templates/app/assets/stylesheets/spree/frontend/components/products/_product_page.scss

```diff
@@ -29,3 +29,50 @@
     }
   }
 }
+
+// Copied from Solidus Demo 3c11395ef6bfda0b906f68dbd11ab5ff567da747
```

Comment:
I'd remove these comments, not sure why it's needed (and we can move this information in the PR description, a git blame should be enough to retrieve them).

Reply:
* I'd like to record where the code came from, lest people think I wrote it by myself lolz :) I think it might be best to record that in the commit message. TODO.
* I feel uncomfortable leaving the copied CSS unmerged with the existing CSS because without the comment, it won't be immediately obvious why they are separate. However, I realize that leaving the comment is a half-measure also, because in order to maintain the correctness of the comment, one would have to avoid editing the copied CSS :p So it looks like the best solution is to merge the existing and copied CSS in this PR. I'll look into it. TODO.